### ISSUE TITLE
Error when activating Python environment

### ISSUE THREAD

**Issue by asavaritayal** [Azure/azure-functions-core-tools#451]
This is from @priyaananthasankar 

When I run `func init . --worker-runtime python`
 
I get this:
 
`Python 3.6 is required. Current python version is '`
 
```bash
(functions_venv) prananths-MacBook-Pro:Serverless prananth$ python --version
Python 3.6.4 :: Anaconda, Inc.
```
 
I have an existing anaconda distribution, but I did activate the virtual environment. It should pick it up from the Venv.

**Comment by asavaritayal** [comment]
@ahmelsayed

**Comment by ahmelsayed** [comment]
fixed in 1f0633737dde7781ae5b99695cf1bca5be38d1d8
